### PR TITLE
Documentation changes for Funnel connector version 4.

### PR DESCRIPTION
**Pull request by thorirfun**

### CODE REVIEW

**Comment by thorirfun** [comment]
> @microsoft-github-policy-service agree [company="{Funnel}"]

**Review by cmcclister — APPROVED** [review]

**Comment by cmcclister** [comment]
@DougKlopfenstein @bezhan-msft I reviewed and approved this PR, but I have no option to merge it. Please advise on how I should proceed. Thanks!

File: powerquery-docs/Connectors/Funnel.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
 ---
 title: Power Query Funnel connector
-description: Provides basic information, prerequisites, and information on how to connect to your Funnel data, along with a list of known issues and limitations.
-author: bezhan-msft
+description: Provides basic information, prerequisites, and information on how to connect to your data, along with a list of known issues and limitations.
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/Funnel.md:3]: Why was "Funnel" removed here?

File: powerquery-docs/Connectors/Funnel.md [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+author: Funnel
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/Funnel.md:4]: This should remain bezhan-msft _or_ a GitHub username from Funnel (not Funnel itself)

File: powerquery-docs/Connectors/Funnel.md [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
+
+
 ms.topic: conceptual
-ms.date: 6/24/2022
-ms.author: bezhan
+ms.date: 03/08/2022
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/Funnel.md:8]: Why is this date moving backwards? This should be 1/11/2023 or greater

File: powerquery-docs/Connectors/Funnel.md [hunk 4]
```diff
@@ -1,13 +1,17 @@
...
+ms.author: Funnel
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/Funnel.md:9]: This needs to remain bezhan

File: powerquery-docs/Connectors/Funnel.md [hunk 5]
```diff
@@ -16,15 +20,16 @@ ms.author: bezhan
 
 | Item | Description |
 | ---- | ----------- |
-| Release State | Beta |
-| Products | Power BI (Datasets)<br/>Power BI (Dataflows) |
-| Authentication Types Supported | Funnel Account |
+| Release State | General Availability |
+| Products | Power BI (Datasets) |
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/Funnel.md:24]: Why was Dataflows removed?

File: powerquery-docs/Connectors/Funnel.md [hunk 6]
```diff
@@ -16,15 +20,16 @@ ms.author: bezhan
...
+| Authentication Types Supported | Funnel Workspace |
+| | |
 
 ## Prerequisites
 
-To use the Funnel connector, you need a Funnel subscription. Funnel helps you collect data from all your marketing platforms, transform it, and send it to the destinations you want, like Power BI (https://funnel.io/).
+To use the Funnel connector, you need a Funnel subscription. Funnel helps you collect data from all your marketing platform, transform it, and send it to the destinations you want, like Power BI (<https://funnel.io/>).
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/Funnel.md:30]: Shouldn't this be "platforms"

File: powerquery-docs/Connectors/Funnel.md [hunk 7]
```diff
@@ -35,34 +40,29 @@ In the Funnel App, go to your account, navigate to the Microsoft Power BI page i
 To connect to Funnel:
 
 1. Select Online Services, find **Funnel** from the product-specific data connector list, and then select **Connect**.
+   ![Select Funnel Connector.](./media/funnel/funnel_connector.png)
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/Funnel.md:43]: Why was lightbox removed?

File: powerquery-docs/Connectors/Funnel.md [hunk 8]
```diff
@@ -35,34 +40,29 @@ In the Funnel App, go to your account, navigate to the Microsoft Power BI page i
...
    > [!NOTE]
-   > The default number of months is 12. If today is 22.03.2022, then you'll get data for the period 01.04.2021 through 22.03.2022.
+   > The default number of months is 12. If today is 22.03.2022 then you will get data for the period 01.04.2021 - 22.03.2022.
 
    You can then either select **Load** to load the data or select **Transform Data** to transform the data.
 
 ## Known issues and limitations
 
-* If a view contains too much data for one month, the connector will get a TOO_MANY_ROWS error.
+* If a Data Share contains too much data for one month the connector will get a TOO_MANY_ROWS error.
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/Funnel.md:66]: Why was this comma removed?

**Review by bezhan-msft — CHANGES_REQUESTED** [review]

File: powerquery-docs/Connectors/Funnel.md (hunk 1 above)
Comment (thorirfun) [reply powerquery-docs/Connectors/Funnel.md:3]: Funnel was removed seeing as it is not Funnel data. The data is in fact the customers data from 3rd parties.

File: powerquery-docs/Connectors/Funnel.md [hunk 9]
```diff
@@ -35,34 +40,29 @@ In the Funnel App, go to your account, navigate to the Microsoft Power BI page i
...
+   ![Successfully signed in to your Funnel Workspace.](media/funnel/funnel_sign_in_3.png)
 
-   For each view, you can enter number of rolling months of data you want.
+1. In the **Navigator** dialog box, choose one or more **Data Shares** from your **Workspaces** to import your data.
+   ![Choose "Data Shares" from Workspaces.](media/funnel/funnel_navigation_table.png)
 
+   For each Data Share you can enter number of rolling months of data you want.
    > [!NOTE]
-   > The default number of months is 12. If today is 22.03.2022, then you'll get data for the period 01.04.2021 through 22.03.2022.
+   > The default number of months is 12. If today is 22.03.2022 then you will get data for the period 01.04.2021 - 22.03.2022.
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/Funnel.md:60]: nit: MSFT style guide uses contractions.

File: powerquery-docs/Connectors/Funnel.md [hunk 10]
```diff
@@ -35,34 +40,29 @@ In the Funnel App, go to your account, navigate to the Microsoft Power BI page i
...
 
    You can then either select **Load** to load the data or select **Transform Data** to transform the data.
 
 ## Known issues and limitations
 
-* If a view contains too much data for one month, the connector will get a TOO_MANY_ROWS error.
+* If a Data Share contains too much data for one month, the connector will get a TOO_MANY_ROWS error.
 
-For more information, read our [Power BI knowledge base articles](https://help.funnel.io/en/?q=powerbi).
+For more information, please read our [Power BI knowledge base articles](https://help.funnel.io/en/?q=powerbi).
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/Funnel.md:68]: nit: remove please

**Review by bezhan-msft — APPROVED** [review]

**Comment by thorirfun** [comment]
Hi! 
Is there anything left for us to do on this PR before this is merged?

**Pull request merged**